$OpenBSD$

Use OpenBSD-style default configuration and log directories for smp-server.

--- apps/smp-server/Main.hs.orig
+++ apps/smp-server/Main.hs
@@ -9,6 +9,6 @@
 defaultCfgPath :: FilePath
-defaultCfgPath = "/etc/opt/simplex"
+defaultCfgPath = "/etc/simplex/smp-server"
 
 defaultLogPath :: FilePath
-defaultLogPath = "/var/opt/simplex"
+defaultLogPath = "/var/simplex/smp-server"
 
